torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -1,3 +1,5 @@
+# mypy: allow-untyped-defs
```

Comment:
we shouldn't do this unless absolutely necessary?
If it's for `MLPModule` and `MLPStacked`, you can use common ancestor `nn.Module` to denote.